web/mocklogin/vite-plugin.ts

```diff
@@ -108,23 +110,25 @@ function buildPlugin(): Plugin {
 
           case "/.hub/local-mock/finish":
```

Comment:
@da-tanabe is this just a code path for local testability but would never be hit for the live one?

Reply:
yup correct… I put `mock` in the path to make that clear